json/stage-json/foodPantry.json

```diff
@@ -4676,7 +4676,7 @@
           "tableView": false,
           "clearOnHide": false,
           "validate": {
-            "custom": "(function () {\n  // check if not a valid character\n  if (/[|;$%\"<>{}+/]/.test(input)) {\n    valid = 'Please do not use special characters e.g. not any of these | ; $ % \" < >{ } + /';\n    return;\n  }\n\n  valid = true;\n})();"
+            "custom": "(function () {\n  // check if not a valid character\n  if (/[|&;$%\"<>(){}+]/.test(input)) {\n    valid = 'Please do not use special characters e.g. not any of these | ; $ % \" < >{ } +';\n    return;\n  }\n\n  valid = true;\n})();"
```

Comment:
Brackets and ampersand have been added to the regex but not to the validation message

Reply:
Now resolved